benchmarking/mod/benchmark.py

```diff
@@ -175,6 +175,7 @@ def run(self):
                     common.pdsh(user, [node], "cd %s; blktrace -d %s -o `hostname`_journal_%s 2>&1 > %s/`hostname`_blktrace_%s.log" % (dest_dir, journal_dev, journal_dev.replace("/","_"), dest_dir, journal_dev.replace("/","_")), option="force")
         if "fatrace" in self.cluster["collector"]:
             common.printout("LOG","Start fatrace data collector under %s " % nodes)
+            time.sleep(15)
```

Comment:
In my environment, some connection error always occurs if using fatrace without sleep here.